contributors_guide.rst

```diff
@@ -0,0 +1,376 @@
+                          4.81939374, 4.75851962])
+            vad_direction = ([359.84659496, 359.30240553, 358.58658589,
+                              357.81073051, 357.01353486])
+            u_wind = ([0.01335138, 0.06014712, 0.12039762,
+                       0.18410404, 0.24791911])
+            v_wind = ([-4.98663937, -4.9398407, -4.87958641,
+                       -4.81587601, -4.75205693])
+
+            vad = pyart.retrieve.velocity_azimuth_display(test_radar,
+                                                          velocity,
+                                                          z_start, z_end,
+                                                          z_count)
+
+            assert_almost_equal(vad.height, vad_height, 3)
+            assert_almost_equal(vad.speed, vad_speed, 3)
+            assert_almost_equal(vad.direction, vad_direction, 3)
+            assert_almost_equal(vad.u_wind, u_wind, 3)
+            assert_almost_equal(vad.v_wind, v_wind, 3)
+
+To install nose::
+
+   		conda install nose
+
+To run all tests in pyart with nose::
+
+		nosetests --exe pyart
+
+All test with in depth details::
+
+		nosetests -v -s
+
+Just one file::
+
+		nosetests filename
+
+
+GitHub
+======
+
+When contributing to pyart, the changes created should be in a new branch
+under your forked repository. Let’s say your adding a new map display.
+Instead of creating that new function in your master branch. Create a new
+branch called ‘cartopy_map’. If everything checks out and the admin
+accepts the pull request, you can then merge the master branch and
+cartopy_map branch. 
+
+To delete a branch both locally and remotely, if done with it::
+
+		git push origin --delete <branch_name>
+		git branch -d <branch_name>
+
+or in this case::
+		
+		git push origin --delete cartopy_map
+		git branch -d cartopy_map
+
+
+To create a new branch::
+                git checkout -b <branch_name>
+
+Typing `git status` will not only inform you on what files have been modified
+and untracked, it will also inform the user of which branch they are on.
+
+To switch between branches, simply type::
+
+		git checkout <branch_name>
+
+When commiting to GitHub, start the statement with a acronym such as
+‘ADD:’ depending on what your commiting, could be ‘MAINT:’ or
+‘BUG:’ or more. Then following should be a short statement such as
+“ADD: Adding cartopy map display.”, but after the short statement, before
+finishing the quotations, hit enter and in your terminal you can then type
+a more in depth description on what your commiting. 
+
+If you would like to type your commit in the terminal and skip the default
+editor::
+
+	git commit -m "PEP: Removing whitespace from vad.py."
+
+To use the default editor(in Linux, usually VIM), simply type::
+
+	git commit
+
+One thing to keep in mind is before doing a pull request, update your
+branches with the original upstream repository.
+
+This could be done by::
+
+	git fetch upstream
```

Comment:
`git merge upstream/master` is also needed to pull in the changed.  I use `git merge --ff-only upstream/master` to prevent a merge commit being made.